tests/test_ssa_generation.py

```diff
@@ -702,10 +626,8 @@ def test_initial_version_exists_for_state_variables_function_assign():
         assert assign.lvalue in phi.rvalues
 
 
-@pytest.mark.skipif(
-    not valid_version("0.4.0"), reason="Solidity version 0.4.0 not available on this platform"
-)
-def test_return_local_before_assign():
+@pytest.mark.skip_unsupported_platforms("0.4.0")
+def test_return_local_before_assign(slither_from_source):
```

Comment:
I'm not sure why this is failing on windows bc it skips the other tests marked `@pytest.mark.skip_unsupported_platforms("0.4.0")`